tests/pypy_bugs.txt

```diff
@@ -4,36 +4,72 @@
 
 broken_exception
 bufaccess
-memoryview
-memslice
+memoryview.memoryview
```

Comment:
This works fine for excluding. It's causing a slight problem in the secondary (non-required) test that attempts to run through the bugs: I can't find a way to include `memoryview/memoryview.pyx` but not the segfaulty test `memoryview/memoryview_in_subclasses.py`

Reply:
Don't regexes work here? E.g. `memoryview.memoryview$`

Reply:
They don't. If I use a regex then the test is no longer excluded correctly.

The trouble is that FileListExcluder is pretty crude and just works on exact string matches

https://github.com/cython/cython/blob/a033f8f63180d1625145bde85e0c5eccf69cd52d/runtests.py#L1880-L1885

I guess the thing to do change that so that it uses the same matching as the test picker.

Reply:
Made this change - I kind of expect it to have some unwanted side effects but I'll look over them once it's run.